src/pages/Admin/VenueList/VenueList.tsx

```diff
@@ -13,7 +14,8 @@ const VenueList: React.FC<VenueListProps> = ({
   selectedVenueId,
   roomIndex,
 }) => {
-  const venues = useSelector((state) => state.firestore.ordered.venues);
+  // @debt This selector relies on all venues in firebase being loaded into memory.. not very efficient
+  const venues = useSelector(orderedVenuesSelector);
```

Comment:
`src/pages/Admin/VenueList/VenueList.tsx` doesn't appear to be used anywhere anymore.

Reply:
OOS, will handle this in a followup PR.